source/help/getting-started/setting-your-status-availability.rst

```diff
@@ -26,7 +26,12 @@ Set a custom status to add an optional emoji to a descriptive status message. Ot
 Clearing a Custom Status
 ------------------------
 
-To clear a custom status, select your avatar at the top of the channel sidebar, then select **Clear Status**, or select the **Clear** option next to your current status.
+By default, new custom statuses aren't removed until they are replaced or cleared manually. When creating a custom status, you can set when it expires by selecting from available options, or by specifying the date and time to clear the status.
```

Comment:
By default, they clear after a day I believe? Or at least that should be the case if you haven't ever touched the "clear after" time. You may want to confirm with a new account

Reply:
Confirmed that the default expiry option is now "Today" rather than "Don't clear". Need to update accordingly.